lldb/source/Symbol/CompilerType.cpp

```diff
@@ -769,19 +771,20 @@ CompilerType::GetBasicTypeFromAST(lldb::BasicType basic_type) const {
 }
 // Exploring the type
 
-std::optional<uint64_t>
+llvm::Expected<uint64_t>
 CompilerType::GetBitSize(ExecutionContextScope *exe_scope) const {
   if (IsValid())
     if (auto type_system_sp = GetTypeSystem())
       return type_system_sp->GetBitSize(m_type, exe_scope);
-  return {};
+  return llvm::createStringError("invalid type; cannot determine size");
```

Comment:
```suggestion
  return llvm::createStringError("invalid type: cannot determine size");
```
We generally concatenate errors with `:` rather than `;`. Will require updating the two tests below.